tests/cow_protocol/ethereum/cow_protocol_ethereum_assert_batches.sql

```diff
@@ -0,0 +1,14 @@
+
+with batches_with_trade_count as (
+    select
+        s.evt_tx_hash,
+        s.evt_block_time,
+        count(t.evt_index) as num_trades
+    from gnosis_protocol_v2_ethereum.GPv2Settlement_evt_Trade t
```

Comment:
can you reference these tables as sources?

Reply:
Yes absolutely.